canvas_modules/harness/cypress/e2e/canvas/drag.cy.js

```diff
@@ -67,6 +67,8 @@ describe("Test to see if regular selection and drag behavior works " +
 
 		// Try dragging a node that is not selected -
 		// this should select the node being dragged and deselect the three selections
+		// TODO -- Fix when autoselect is available.
```

Comment:
For each place where we added this comment please add another comment line underneath each one saying:
```
// See: https://github.ibm.com/NGP-TWC/wdp-abstract-canvas/issues/3760
```